Written standups post (#11)
bobek/bobek.cz: source/blog/2021/written-standup.html.md

- On the other hand, when you start working in the morning. Quickly go through the stand up from the **previous day**. That will help you to see what others have done.
- Always provide context to allow people to understand the update (links to Phabricator tasks, GitLab projects, merge requests or just link what ever you are talking about).
- If somebody asked a question / requested more details via comment, always consider updating your record in the document (in contrast to just replying to the comment). Any consecutive reader will benefit from the updated text.

Important thing is to keep common sense in the play. Cover information, which could be interesting for others. Not wasting time on trivial tasks. Always keeping on mind, that somebody else is going to read it.

![Example of a written standup](standup_example.png) *Example page from a written standup document. Covering an IoT project.*

## Key benefits of written standups

> - One can read way faster then listen/speak. So you can cover way more people when reading their notes.

Reviewer: ```suggestion
- One can read way faster than listen/speak. So you can cover way more people when reading their notes.
```

- Having a links to additional resources give opportunity to dig a bit deeper and ask meaningful questions.
- Google Document allows for inline comments which became a great tool for asking additional information asynchronously.
- It is fair for everybody, despite their schedules or time-zones. It turned to be really valuable for every team, we have rolled this out to. It works for team in one time-zone as well as for teams distributed around the globe. Standup basically follows the sun.

> - Allows people, who are for any reason were missing on a particular standup, to catch up easily.

Reviewer: ```suggestion
- Allows people, who for any reason missed a particular standup, to catch up easily.
```

- Produces body of *searchable* and *rich* documents.
- It also helps you to see what you have achieved during the day. And get the sense of accomplishment, which is otherwise challenging to get at knowledge jobs.
- Best is to treat written standup as your logbook. Filling it during the day. It removes burden of remembering what you have done at the end of the day. As well as it allows for faster pickup by others.

## Adoption

> Current experience is, that people adopt the concept very quickly.

Reviewer: I have hard time with commas :-) This one feels strongly out of place though.

```suggestion
Current experience is that people adopt the concept very quickly.
```
